### ISSUE TITLE
test-cdk validate_network_policies failure in 1.10

### ISSUE THREAD

**Issue by kwmonroe** [juju-solutions/kubernetes-jenkins#161]
`test_deploy` failed in `validate_network_policies` for cdk-canal using beta charms and 1.10/edge snaps:

https://ci.kubernetes.juju.solutions/job/test-cdk/502/testReport/junit/(root)/test_cdk/test_deploy_canonical_kubernetes_canal_/

```
        query_from_good = "/snap/bin/kubectl exec bboxgood -n netpolicy -- wget --timeout=3  nginx.netpolicy"
        cmd = await unit.run(query_from_good)
        assert cmd.status == 'completed'
>       assert "index.html" in cmd.data['results']['Stderr']
E       assert 'index.html' in 'Error from server (NotFound): pods "bboxgood" not found\n'
```

**Comment by battlemidget** [comment]
I've not run into this error recently, do we think this is still an issue?

**Comment by kwmonroe** [comment]
@battlemidget, this was fixed with a combination of #162 from @hyperbolic2346 and the retry tweaks from @ktsakalozos  (https://github.com/juju-solutions/kubernetes-jenkins/commit/c8ced7cbcaba116b1741e28e8b246172e010bdb7#diff-afe81d09d13832860b6f7f81f15ea585R318)... both of which are now relics of a jenkins long forgotten.
